### DISCUSSION TITLE
Blender failed to start on termux

### DISCUSSION THREAD

**Discussion by pawn343** [termux-user-repository/tur#724] [Q&A]
First of all i really surprised knowing that blender is actually on termux right now.
But, when i've installed blender and wanted to start it by using termux-11 it's failed with the error:
```
CANNOT LINK EXECUTABLE "blender": library "libIex-3_1.so" not found
```

And im looking up the file with `apt-file find "libIex"` and found that mine is `libIex-3_2.so`.

So, i wondering what should i do now?

**Comment by T-Dynamos** [comment]
I am in charge of fixing this error, I will update blender soon.

**Comment by Basthet** [comment]
Just wondering - what happened to blender in the repo? 

Error: Client error '404 Not Found' for url 
'https://github.com/termux-user-repository/dists/releases/download/0.1/blender_1.3.6.11_aarch64.deb'

can it be added again?

**Reply by T-Dynamos** [reply]
CC: @licy183 

```
~$ apt download blender

Err:1 https://tur.kcubeterm.com tur-packages/tur-on-device aarch64 blender aarch64 1:3.6.11
  404  Not Found [IP: 20.207.73.82 443]
E: Failed to fetch https://github.com/termux-user-repository/dists/releases/download/0.1/blender_1.3.6.11_aarch64.deb  404  Not Found [IP: 20.207.73.82 443]
```
